src/main/java/janggi/point/Point.java

```diff
@@ -0,0 +1,52 @@
+package janggi.point;
+
+public record Point(int row, int column) {
+
+    private static final int BOARD_LOWER_BOUND = 0;
+    private static final int BOARD_ROW_UPPER_BOUND = 9;
+    private static final int BOARD_COLUMN_UPPER_BOUND = 8;
+
+    public Point(int row, int column) {
```

Comment:
```suggestion
    public Point {
```
record로 선언해서 불필요해보이네요.

Reply:
제거했습니다. :)